packages/workbox-precaching/src/PrecacheController.ts

```diff
@@ -15,12 +15,11 @@ import {WorkboxPlugin} from 'workbox-core/types.js';
 
 import {cleanRedirect} from './utils/cleanRedirect.js';
 import {createCacheKey} from './utils/createCacheKey.js';
+import {PrecacheEntry} from './_types.js';
```

Comment:
Super nit, but BTW my preference has been to order imports alphabetically by their filename rather than their import names. My reasoning: when you import multiple symbols (or import X as Y), it's no longer possible to maintain an alphabetical ordering by import name.

Reply:
I'm fine with that, but I'd prefer to see the `import './_version.js';` stand on its own, and not mix in "meaningful" imports with it.

So the `'./_types.js';` import would show up first in the list of "local" imports, separate from the versioning import.

I just installed https://marketplace.visualstudio.com/items?itemName=Snowcoders.sci-sort-import-groups to help automate this in the future.

Reply:
Yeah, that's fine. I think it was out of order before as well anyway...